config/jobs/kubernetes-csi/external-attacher/external-attacher-config.yaml

```diff
@@ -377,6 +377,16 @@ presubmits:
       containers:
       # We need this image because it has Docker in Docker and go.
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230105-c1beb6cf98-master
+        resources:
+          requests:
+            # these are both a bit below peak usage during build
+            # this is mostly for building kubernetes
+            memory: "9000Mi"
+            # during the tests more like 3-20m is used
+            cpu: 2000m
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
```

Comment:
Does this block have to be moved up from line 402?  Similar comments to below changes as well.

Reply:
Well the main reason I did this is because of the changes to the gen-jobs.sh script. `VOLUME_MODE_CONVERSION_TESTS` needs to be added only for external-provisioner, so there's another if-else block at the end of the env section.

Reply:
Sure

Reply:
That's fine.  Can you add comments here so that others don't move it by mistake and break the script?

Reply:
Done!

Reply:
The comment now appears in the generated YAML files which confused me because it makes no sense in that context.

See my comment on the script.